src/arfs/arfs_builders/arfs_file_builders.ts

```diff
@@ -112,13 +113,13 @@ export class ArFSPublicFileBuilder extends ArFSFileBuilder<ArFSPublicFile> {
 export class ArFSPrivateFileBuilder extends ArFSFileBuilder<ArFSPrivateFile> {
 	cipher?: string;
 	cipherIV?: CipherIV;
+	fileKey?: FileKey;
 
 	constructor(
 		readonly fileId: FileID,
 		readonly arweave: Arweave,
 		private readonly driveKey: DriveKey,
-		readonly owner?: ArweaveAddress,
-		readonly fileKey?: FileKey
+		readonly owner?: ArweaveAddress
```

Comment:
Add `fileKey` back again
```suggestion
		readonly owner?: ArweaveAddress,
		readonly fileKey?: FileKey
```